.circleci/config.yml

```diff
@@ -254,11 +254,13 @@ jobs:
           command: yarn db:seed:ci
       - run:
           name: Test backend
-          command: yarn test:ci
-      - store_test_results:
-          path: reports/
-      - store_artifacts:
-          path: coverage/
+          command:  |
+            chmod 744 ./bin/test-backend-ci
+            ./bin/test-backend-ci
+      # - store_test_results:
+      #     path: reports/
+      # - store_artifacts:
+      #     path: coverage/
```

Comment:
Do we have a way of accessing these artifacts?

Reply:
I've been using a bash script to fetch all coverage files, but I think I'd prefer to just pass lcov.info to genhtml locally as we just talked about on slack.

fwiw this is that script

```bash
#!/usr/bin/env bash

read -p "Enter build number: " BUILD_NUMBER

function path_maker() {
  PATH_TO_MAKE=`echo $1 | sed 's/.*\/coverage\///' | sed 's/\/[^\/]*$//'`
  mkdir -p $PATH_TO_MAKE
  echo $PATH_TO_MAKE
}

function get_filename_from_url() {
  echo $1 | sed 's/.*\/coverage\///' | sed 's/\//\n/g' | tail -n 1
}

function downloader() {
  # read artifacts.txt which is a list of URLs to the coverage reports.
  while read -r line; do
    wget --verbose --header "Circle-Token: <CI_TOKEN>" $line -O $(path_maker $line)/$(get_filename_from_url $line)
  done < artifacts.txt
}

curl -s -H "Circle-Token: <CI_TOKEN>" https://circleci.com/api/v1.1/project/github/HHS/Head-Start-TTADP/$BUILD_NUMBER/artifacts | jq -r '.[].url'  > artifacts.txt
downloader
```

Reply:
This would be a cool thing to add to our scripts, especially if you had the time to write up a brief instruction for it